spacy_llm/models/integration/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .remote.base import Remote
+from .remote.langchain import model_langchain, query_langchain
+from .remote.minichain import model_minichain, query_minichain
+
+__all__ = [
+    "Remote",
+    "query_minichain",
+    "query_langchain",
+    "model_minichain",
+    "model_langchain",
+]
```

Comment:
Do I remember correctly that we had removed this `integration` subdir? (merge conflict leftover?)

Reply:
Yes, seems to have been a merge slip-up.

Reply:
Removed in https://github.com/explosion/spacy-llm/pull/179/commits/88b2c8f2aeb78242bf26304caefae918b9478c12.